doc/explanation/microcloud.rst

```diff
@@ -64,6 +64,18 @@ MicroCloud will still be usable, but you will see some limitations:
   As a result of this, network forwarding works at a basic level only, and external addresses must be forwarded to a specific cluster member and don't fail over.
 - There is no support for hardware acceleration, load balancers, or ACL functionality within the local network.
 
+Dedicated networks for Ceph
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
+You can set up dedicated networks for Ceph to differentiate between public and internal Ceph traffic.
+
+- You can choose to have a dedicated network for the public Ceph traffic and another dedicated network for the Ceph internal (cluster) traffic. This is a fully disaggregated Ceph network setup.
```

Comment:
This could be rephrased to something like as this seemed overlay verbose when I read it:

> You can choose to separate Ceph public and internal traffic onto separate dedicated networks (fully disaggregated) or you can choose to have all Ceph traffic use a single separate network (partially disaggregated).
> In either setups all of your cluster members must have an IP address configured on a network interface for the Ceph network(s) specified.

Reply:
fixed